packages/locale/src/address.mock.ts

```diff
@@ -0,0 +1,19 @@
+import { Address } from '@bigcommerce/checkout-sdk';
+
+export function getAddress(): Address {
```

Comment:
payment-integration-test-utils package already has this mock, no need to duplicate it, better import it from there

Reply:
This package is located in checkout-sdk project but not here.